k8s/kind/setup.sh

```diff
@@ -1,3 +1,3 @@
-docker image build -t fwnetworking/kindnode:latest -f Dockerfile .
+docker image build -t localhost:5000/kindnode:latest -f Dockerfile .
```

Comment:
Just for my info: Is this correct; having "localhost:5000" in the image name? Does it matter? And if so, shouldn't it get the value of $DOCKER_ACC ?

Reply:
Yeah, for local and private docker registry, instead of prefixing with the dockerhub repo name, we preface the image name with the registry name.

You're right, I'll change it to $DOCKER_ACC

Reply:
Actually, it looks like this file was an old precursor to kind-setup.sh
We don't actually use it anymore, I will go ahead and delete it.